ui/src/pages/transformations/lookup_product_item.js

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright (c) 2023, CloudBlue LLC
+All rights reserved.
+*/
+import createApp from '@cloudblueconnect/connect-ui-toolkit';
+import '@fontsource/roboto/500.css';
+import '../../../styles/index.css';
+import '../../../styles/lookup.css';
+import '../../../styles/app.styl';
+import {
+  getLookupProductItemCriteria,
+  validate,
+} from '../../utils';
+import {
+  hideComponent,
+  showComponent,
+  showError,
+} from '../../components';
+
+
+export const createOutputColumnForLookup = (prefix, name) => ({
+  name: `${prefix}.${name}`,
+  type: 'string',
+  description: '',
+});
+
+export const lookupProductItem = (app) => {
+  if (!app) return;
+
+  let columns = [];
+
+  app.listen('config', async (config) => {
+    const {
+      context: { available_columns: availableColumns, stream },
+      settings,
+    } = config;
+
+    const hasProduct = 'product' in stream.context;
+    columns = availableColumns;
+    const criteria = await getLookupProductItemCriteria();
+
+    hideComponent('loader');
+    showComponent('app');
+
+    Object.keys(criteria).forEach((key) => {
+      const option = document.createElement('option');
+      option.value = key;
+      option.text = criteria[key];
+      document.getElementById('criteria').appendChild(option);
+    });
+
+    availableColumns.forEach((column) => {
+      const option = document.createElement('option');
+      option.value = column.id;
+      option.text = column.name;
+      document.getElementById('column').appendChild(option);
+
+      const anotherOption = document.createElement('option');
```

Comment:
Why you did not append ```option``` twice?

Reply:
Tried. It appends only in the first select. In the second one, an empty option is appended.